eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -36,6 +36,14 @@ param (
 )
 eng/common/scripts/Import-AzModules.ps1
 
+# Import resource management helpers and override its Log function.
+. "$PSScriptRoot\..\common\scripts\Helpers\Resource-Helpers.ps1"
+
+function Log($Message) {
```

Comment:
We have some logging helpers at https://github.com/Azure/azure-sdk-tools/blob/main/eng/common/scripts/logging.ps1 that will help with errors as well.

Reply:
Actually did you intentionally override the log method here?

Reply:
I already had some for TestResources and don't want to change the output. All this does is override the `Log` function in _Resource-helpers.ps1_ to log in a way this script is already using.